### PR TITLE
Added Powershell Cmdlets to support Enabling/Disabling GeoBackupPolicy for SQL Azure DataWarehouses

### PR DESCRIPTION
**Pull request by alazad-msft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@alazad-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@akromm @yoavrubin Can you guys take a look at this and sign off if this looks OK and doesn't confliuct with planned changes this release?

**Comment by akromm** [comment]
looks good to me.  What are the planned changes this might conflict with?

File: src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/AzureSqlDatabaseGeoBackupPolicyCmdletBase.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+using Microsoft.Azure.Commands.Sql.Backup.Model;
+using Microsoft.Azure.Commands.Sql.Backup.Services;
+using Microsoft.Azure.Commands.Sql.Common;
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.Azure.Commands.Sql.Database.Services;
+
+namespace Microsoft.Azure.Commands.Sql.Backup.Cmdlet
+{
+    public abstract class AzureSqlDatabaseGeoBackupPolicyCmdletBase : 
+        AzureSqlCmdletBase<IEnumerable<AzureSqlDatabaseGeoBackupPolicyModel>, AzureSqlDatabaseBackupAdapter>
```
Comment (yoavrubin) [inline src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/AzureSqlDatabaseGeoBackupPolicyCmdletBase.cs]: Why aren't you inheriting from AzureSqlDatabaseCmdletBase, it already provides the server and database parameters

Comment (alazad-msft) [inline src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/AzureSqlDatabaseGeoBackupPolicyCmdletBase.cs]: I need a AzureSqlDatabaseBackupAdapter object to be returned with InitModelAdapter, so an override would be needed anyways.

`protected override AzureSqlDatabaseBackupAdapter InitModelAdapter(AzureSubscription subscription)
{
return new AzureSqlDatabaseBackupAdapter(DefaultProfile.Context);
}`

Comment (yoavrubin) [inline src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/AzureSqlDatabaseGeoBackupPolicyCmdletBase.cs]: Discussed offline, @alazad-msft will update the inheritance

**Comment by yoavrubin** [comment]
LGTM

**Comment by cormacpayne** [comment]
@azuresdkci test this please

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: ChangeLog.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## 2016.08.09 version 2.0.1
```
Comment (markcowl) [inline ChangeLog.md]: @alazad-msft Please revert changes to this file

Comment (alazad-msft) [inline ChangeLog.md]: @markcowl Reverted ChangeLog.md

**Comment by cormacpayne** [comment]
@alazad-msft Hey Alec, the build is failing because you are missing help for the two cmdlets that you have added.

**Comment by yoavrubin** [comment]
In this PR https://github.com/Azure/azure-powershell/pull/2824 I've made a change that affects all of AzureSQL's cmdlets. Please take a look there and make the relevant changes to your new cmdlets (these are changes that related to using ConfirmAction).

**Comment by alazad-msft** [comment]
@cormacpayne Thanks! I noticed it as well. I have made the needed updates and resolved the issues. Please continue with merge.

File: src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseGeoBackupPolicy.cs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+using Microsoft.Azure.Commands.Sql.Database.Model;
+
+namespace Microsoft.Azure.Commands.Sql.Backup.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to create or update a new Azure Sql Database geo backup policy
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, "AzureRmSqlDatabaseGeoBackupPolicy",
+        SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.Low)]
```
Comment (markcowl) [inline src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseGeoBackupPolicy.cs]: Why confirmimpact.low?  Normally, cmdlets that make changes would have the default confirmpmact (confirmimpact.medium)

**Comment by markcowl** [comment]
@alazad-msft One nit + please pull (and rebase on using ```git pull --rebase``` the latest changes, as these include a test fix).

File: src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseGeoBackupPolicy.cs (hunk 3 above)
Comment (alazad-msft) [inline src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseGeoBackupPolicy.cs]: @markcowl Fixed - upgraded set cmdlet to ConfirmImpact.Medium.

**Comment by alazad-msft** [comment]
@markcowl Fixed nit and rebased. Thanks for your kind review, Mark.

**Comment by cormacpayne** [comment]
On-demand: http://azuresdkci.cloudapp.net/job/powershell-demand/1116/

**Pull request merged**